.github/workflows/scheduled-sync.yml

```diff
@@ -4,7 +4,7 @@ name: Sync Data on Schedule
 
 on:
   schedule:
-  - cron: '0 */12 * * *'
+  - cron: '17 * * * *'
```

Comment:
Once every hour on the 17th minute. 17 chosen to offset high demand for actions pipelines.